### ISSUE TITLE
stolen

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11455]
|<img alt="commons-dev-subscribe's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| commons-dev-subscribe reported|
|-|-|
|Trac ID|trac#561|
|Type|enhancement|
|Created|2004-03-23 15:27:12Z|

Attachments:

* [me.zip](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/77c/77c8184f671aa0397dd897541ed5ec0a8be0380b/26424d23001756d8d8589f52a6aa7cf0e8802b90.zip) (22126 bytes) - added by commons-dev-subscribe on 2004-03-23 15:27:12Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__561 561
type__enhancement enhancement
reporter__commons_dev_subscribe commons-dev-subscribe
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1080055632000000 1080055632000000
changetime__1477914682886966 1477914682886966
version__ 
owner__ 
cc__commons-dev-subscribe
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="commons-dev-subscribe's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|commons-dev-subscribe commented|
|-|-|

```
#!html
<pre>
misc
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Looks like a spam/virus.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

In changeset 48dfa7756c8bf0221658eba814f7c5ce36396310
```
#!CommitTicketReference repository="" revision="48dfa7756c8bf0221658eba814f7c5ce36396310"
Merge pull request #561 from twisted/6320-rodrigc-irc-py3-2

Author: rodrigc
Reviewer: acabhishek942
Fixes: #6320

Port twisted.words.protocols.irc to Python 3
```
